### PR TITLE
Add app_reviews templates

### PR DESCRIPTION
**Pull request by gchhablani**
**Notes:**
- I am not sure about `convert_to_star_rating`. Should a model be able to generate emojis? 🤔

**Edit**
This is from `bert-base-uncased`:
```python
>>> bt('[UNK]')
{'input_ids': [101, 100, 102], 'token_type_ids': [0, 0, 0], 'attention_mask': [1, 1, 1]}
>>> bt('🤔')
{'input_ids': [101, 100, 102], 'token_type_ids': [0, 0, 0], 'attention_mask': [1, 1, 1]}
```

### CODE REVIEW

File: templates/app_reviews/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+      |||
+
+      {{review}}"
+    name: generate_review
+    reference: Generate a review from the rating.
+  9746ce4b-ac58-4dfb-9783-d77c95cb62cf: !Template
+    id: 9746ce4b-ac58-4dfb-9783-d77c95cb62cf
+    jinja:
+      "What would be the star rating of this review?\n|||{% if star==5 %}\n\u2B50\
```
Comment (craffel) [inline templates/app_reviews/templates.yaml:47]: This prompt does not have the review in it.

File: templates/app_reviews/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      \u2B50\u2B50\u2B50\u2B50\n{% elif star==4%}\n\u2B50\u2B50\u2B50\u2B50\n{% elif\
+      \ star==3%}\n\u2B50\u2B50\u2B50\n{% elif star==2%}\n\u2B50\u2B50\n{% else %}\n\
+      \u2B50\n{% endif %}"
+    name: convert_to_start_rating
+    reference: Given the review, generate a star rating.
+  d34e1413-2699-4701-baa2-05d931d012ba: !Template
+    id: d34e1413-2699-4701-baa2-05d931d012ba
+    jinja: "What would be the star rating of this review: {{review}}?
```
Comment (craffel) [inline templates/app_reviews/templates.yaml:55]: This prompt and the one above are ambiguous - the model won't know whether you mean for it to generate the number of stars, a certain number of emojis, etc. I think you need to give the model more instructions as to which you want.

**Review by craffel — CHANGES_REQUESTED** [review]

File: templates/app_reviews/templates.yaml (hunk 2 above)
Comment (gchhablani) [reply templates/app_reviews/templates.yaml]: I've updated the rating information for both cases. I've added emojis for the star-case, hope that is fine?

File: templates/app_reviews/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+      |||
+
+      {{star}}"
+    name: convert_to_rating
+    reference: Convert review to rating
+  f923aeeb-1150-486d-99cd-c4ab93ed5130: !Template
+    id: f923aeeb-1150-486d-99cd-c4ab93ed5130
+    jinja:
+      "Would you recommend an app to a friend which has a {{star}}-star rating on a scale of 1 to 5 (1 being lowest and 5 being highest)?
```
Comment (craffel) [inline templates/app_reviews/templates.yaml:66]: This prompt will only result in 5 unique examples - I'm not sure that's enough, I think the model would just memorize the mapping right away.

File: templates/app_reviews/templates.yaml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+      |||
+
+      {{review}}"
+    name: generate_review
+    reference: Generate a review from the rating.
+  9746ce4b-ac58-4dfb-9783-d77c95cb62cf: !Template
+    id: 9746ce4b-ac58-4dfb-9783-d77c95cb62cf
+    jinja:
+      "What would be the \u2B50-rating of this review (\u2B50 being lowest and \u2B50\u2B50\u2B50\u2B50\u2B50 being highest) : {{review}}?\n|||{% if star==5 %}\n\u2B50\
```
Comment (craffel) [inline templates/app_reviews/templates.yaml:47]: Looking in to it, T5's vocab can't tokenize the star emoji:
https://colab.research.google.com/drive/1uXD0f2QdRdk9hMTIW9wF2S_R5tvDperz?usp=sharing
Why not just use * instead?

**Review by craffel — CHANGES_REQUESTED** [review]

File: templates/app_reviews/templates.yaml (hunk 3 above)
Comment (gchhablani) [reply templates/app_reviews/templates.yaml:66]: Makes sense. I will remove this template then.

**Comment by craffel** [comment]
Thanks!

**Pull request merged**